backend/src/main/resources/application.properties

```diff
@@ -24,7 +24,7 @@ mp.openapi.extensions.smallrye.operationIdStrategy=CLASS_METHOD
 
 quarkus.http.access-log.enabled=true
 quarkus.http.access-log.category=access_log
-quarkus.http.access-log.pattern=combined
+quarkus.http.access-log.pattern=%h %l %u %t "%r" %s %b "%{i,Referer}" "%{i,User-Agent}" orgId=%{d,x-rh-rbac-org-id}
```

Comment:
`combined` pattern from the [Quarkus doc](https://quarkus.io/guides/http-reference#quarkus-vertx-http-config-group-access-log-config_quarkus.http.access-log.pattern) concatenated with `orgId=%{d,x-rh-rbac-org-id}`.

Reply:
Before:
```
2022-09-21T08:51:28.8959224Z 2022-09-21 08:51:28,889 INFO  [info] (executor-thread-0) 127.0.0.1 - user 21/Sep/2022:08:51:28 +0000 "GET /api/integrations/v1.0/endpoints/9de0632b-d4ab-4e16-9f29-af7ff1b7c87c/history HTTP/1.1" 200 241 "-" "Apache-HttpClient/4.5.13 (Java/11.0.11)"
```
After:
```
2022-09-21T09:30:38.3643979Z 2022-09-21 09:30:38,310 INFO  [info] (executor-thread-0) 127.0.0.1 - user 21/Sep/2022:09:30:38 +0000 "GET /api/integrations/v1.0/endpoints/e69c36c4-8e11-41c4-aa5d-1214957b8f49/history HTTP/1.1" 200 241 "-" "Apache-HttpClient/4.5.13 (Java/11.0.11)" orgId=someOrdId
```